### DISCUSSION TITLE
Tennis competition stage marker (qualification / main stage)

### DISCUSSION THREAD

**Discussion by gone213** [cloudbet/docs#86] [Ideas]
Hi

  It would be great to add a competition stage marker in case we get the odds of upcoming competitions (_**https://sports-api.cloudbet.com/pub/v2/odds/fixtures?sport=tennis&date=2024-07-03&players=false&limit=10000**_).

  At the moment there is no way to determine if it is a **main stage** or a **qualification stage** of a particular tournament - the received key does not contain this marker:
```
{
  "competitions": [
    {
      "name": "Wimbledon, Women Singles",
      "key": "tennis-wta-wimbledon-women-s-singles",
      "sport": {
        "name": "Tennis",
        "key": "tennis"
       ............
    {
      "name": "ATP Challenger Brasov, Men Singles",
      "key": "tennis-challenger-t9a09-atp-challenger-brasov-romania-men-singles",
      "sport": {
        "name": "Tennis",
        "key": "tennis"
      .............  	  
}
```	  

Thank you for your patience ))

**Comment by kgravenreuth** [comment]
Hi @gone213 ,
Thanks for your feedback. While we currently don't provide such information, we will take this feedback under review and let you know if or when we make changes. 

Regards,
Klaus Gravenreuth
Product & Engineering @ Cloudbet
